names.md

```diff
@@ -1,7 +1,7 @@
 subl# This is a list of all the names of people working on this project
 
 # Format -   Full name - nickname (optional)
-Rucha Rathor
+Rucha Rathor-White
```

Comment:
You created a new branch based off your color branch.  You should create a new branch based off master